README.md

```diff
@@ -57,11 +93,10 @@ python3 -m pip install https://aws-mx-pypi.s3-us-west-2.amazonaws.com/1.8.0/aws_
 python3 -m pip install https://aws-mx-pypi.s3-us-west-2.amazonaws.com/1.8.0/aws_mx_cu110-1.8.0-py2.py3-none-manylinux2014_x86_64.whl
 ```
 
-Once you have MXNet, you can install our version of AutoGluon:
+Once you have MXNet, you can install the AutoGluon:
```

Comment:
"the AutoGluon" -> "AutoGluon"

Reply:
I've removed the content about the customized version of AG since it has been merged already.